src/submission/validation-clinical/donor.ts

```diff
@@ -1,53 +1,42 @@
 import {
-  FieldsEnum,
   DataValidationErrors,
   SubmittedClinicalRecord,
   SubmissionValidationError,
   ClinicalInfoFieldsEnum,
-  ValidatorResult,
-  ModificationType,
-  ClinicalEntityType,
+  RecordValidationResult,
+  DonorRecordsOrganizer,
 } from '../submission-entities';
 import { DeepReadonly } from 'deep-freeze';
 import { Donor } from '../../clinical/clinical-entities';
 import * as utils from './utils';
 import _ from 'lodash';
+import { DonorRecordsOrganizerOperations as organizerOperations } from './utils';
 
 export const validate = async (
-  newRecords: DeepReadonly<{ [clinicalType: string]: SubmittedClinicalRecord }>,
+  newRecordsOrganizer: DeepReadonly<DonorRecordsOrganizer>,
```

Comment:
`submittedRecords`

Reply:
no need to mention the organizer in the name we have the type info

Reply:
yea, I had a different setup before where `organizer` made more sense but I changed it, should've updated the naming too